addons/hr_expense/models/hr_expense.py

```diff
@@ -275,6 +277,84 @@ def action_get_attachment_view(self):
         res['context'] = {'default_res_model': 'hr.expense', 'default_res_id': self.id}
         return res
 
+    @api.model
+    def get_empty_list_help(self, help_message):
+        if help_message:
+            alias_record = self.env.ref('hr_expense.mail_alias_expense')
+            if alias_record and alias_record.alias_domain and alias_record.alias_name:
+                dynamic_help = '<p>%s</p>' % _("""Create a new expense, or send receipts by email to %(link)s  to automatically create a new expense. Send an email right now and see it here.""") % {
+                    'link': "<a href='mailto:%(email)s'>%(email)s</a>" % {'email': '%s@%s' % (alias_record.alias_name, alias_record.alias_domain)}
+                }
+                return '<p class="oe_view_nocontent_create">%s</p>%s%s' % (
+                    _('Click to add a new expense'),
+                    dynamic_help,
+                    help_message)
+        return super(HrExpense, self).get_empty_list_help(help_message)
+
+    @api.model
+    def message_new(self, msg_dict, custom_values=None):
```

Comment:
To ensure the balance of space-time continuum, don't forget to get @tde-banana-odoo 's benediction for the override of `message_new` before merging ;)